src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CSharpStatementBlockOnTypeFormattingTest.cs

```diff
@@ -37,6 +37,39 @@ public async Task CloseCurly_IfBlock_MultiLine()
 }
 ",
 expected: @"
+@{
+    if (true)
+    {
+    }
+}
+",
+triggerCharacter: "}");
+        }
+
+        [Fact]
+        public async Task CloseCurly_MultipleStatementBlocks()
```

Comment:
Unfortunately this test doesn't really test the code path that I added because it doesn't use a real formatter.